### DISCUSSION TITLE
[Doc Bug] Metadata Syntax Doc: Specifying docker image on github fails

### DISCUSSION THREAD

**Discussion by lucianposton** [community/community#26180] [Actions]
<p>The documentation at <a href="https://help.github.com/en/actions/building-actions/metadata-syntax-for-github-actions#example-using-public-docker-container-on-github" rel="nofollow noopener">https://help.github.com/en/actions/building-actions/metadata-syntax-for-github-actions#example-using-public-docker-container-on-github</a> indicates that one can use public docker containers packaged on github by specifying one as the image of a github action.</p>
<p>Other posts here on github.community suggest that this feature is planned but not yet implemented. If so, the doc should be fixed accordingly.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/4e777cfc0616f4b8d759f3b6490cbfbd662ed570.png" data-download-href="/uploads/short-url/bc9fbSMs8eKmHMVsinuBg7rb1uM.png?dl=1" title="lol8.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/646121/181095945-299c636e-2846-46fe-829c-e7ce13e27c5f.png" alt="lol8.png" data-base62-sha1="bc9fbSMs8eKmHMVsinuBg7rb1uM" width="690" height="103" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/4e777cfc0616f4b8d759f3b6490cbfbd662ed570_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">lol8.png</span><span class="informations">738×111 6.78 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Following that documentation results in an error during the initial setup phase of a workflow indicating that the specified value for ‘image:’ is incorrect.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/1dcc48d0d871d96e9ae65ba07fa05f102fd78d7c.png" data-download-href="/uploads/short-url/4fBwun3Qirk8vbENTkOC0G7cHVO.png?dl=1" title="lol5.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/646121/181095946-a5e54872-5cef-403e-8384-28277d24b472.png" alt="lol5.png" data-base62-sha1="4fBwun3Qirk8vbENTkOC0G7cHVO" width="690" height="198" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/1dcc48d0d871d96e9ae65ba07fa05f102fd78d7c_2_690x198.png, https://aws1.discourse-cdn.com/github/optimized/1X/1dcc48d0d871d96e9ae65ba07fa05f102fd78d7c_2_1035x297.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/1dcc48d0d871d96e9ae65ba07fa05f102fd78d7c.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/1dcc48d0d871d96e9ae65ba07fa05f102fd78d7c_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">lol5.png</span><span class="informations">1239×356 38.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>If instead you try ‘image: docker://docker.pkg.github.com/$GITHUB_REPOSITORY/name:tag’, the setup phase will succeed, but the docker pull will fail due to an authentication error on github package’s docker server side.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/e084667195b8c0f4276df367a1d4650e013e2efb.png" data-download-href="/uploads/short-url/w2aCp7pQNhLuJDn9lFLOxCzSvuX.png?dl=1" title="lol2.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/646121/181095947-e34357cf-8d0e-4860-b659-efedb93a8b6a.png" alt="lol2.png" data-base62-sha1="w2aCp7pQNhLuJDn9lFLOxCzSvuX" width="690" height="265" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/e084667195b8c0f4276df367a1d4650e013e2efb_2_690x265.png, https://aws1.discourse-cdn.com/github/optimized/1X/e084667195b8c0f4276df367a1d4650e013e2efb_2_1035x397.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/1X/e084667195b8c0f4276df367a1d4650e013e2efb_2_1380x530.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/e084667195b8c0f4276df367a1d4650e013e2efb_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">lol2.png</span><span class="informations">1518×584 82.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by weide-zhou** [comment]
<p>Hi <a class="mention" href="/lucianposton">@lucianposton</a> ,</p>
<p>Thank you for reaching this out! I can reprodude the same when i use 'image: ‘actions/container-template@master’ in my docker action. I have submitted this to github engineering team.  I will notify you once there is any update from them. </p>
<p>Thanks again!</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/lucianposton">@lucianposton</a> ,</p>
<p>Confirmed with Github, the doc is wrong. Using 'image: ‘actions/container-template@master’ in docker action is not supported. Now the content has been removed and fixed in the <a href="https://help.github.com/en/actions/building-actions/metadata-syntax-for-github-actions#example-using-public-docker-container-on-github" rel="nofollow noopener">doc</a>. Please check.</p>
<p>Thanks for pointing this out!</p>
